elfutils/plan.sh

```diff
@@ -6,8 +6,8 @@ pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_description="elfutils is a collection of various binary tools such as
   eu-objdump, eu-readelf, and other utilities that allow you to inspect and
   manipulate ELF files."
-pkg_upstream_url=https://sourceware.org/elfutils/
-pkg_source=https://sourceware.org/${pkg_name}/ftp/${pkg_version}/$pkg_name-$pkg_version.tar.bz2
+pkg_upstream_url=https://fedorahosted.org/elfutils/
```

Comment:
IIRC, I'm the one who changed this to sourceware because the fedorahosted links redirect to sourceware.

Reply:
It builds either way, going ahead and changing it back.

Reply:
Better to go directly to the source than through the redirect.